upstream-tag-on-merge/package.json

```
@@ -32,6 +31,6 @@
     "format:check": "prettier --check \"src/**/*.ts\"",
     "lint:staged": "lint-staged",
     "type-check": "tsc --noEmit",
-    "prepare": "husky"
+    "prepare": "git config core.hooksPath .githooks"
```

ditto:
```suggestion
    "dev:setup": "git config core.hooksPath .githooks"
```
or similar

---

@devnoname120 I would rather stick with `prepare` as it ensures that the hook is installed the same whay as husky is doing it. If we use whatever other script then it would mean you need to manually run it. I don't like this.

---

Not sure if you are both aware since no one has mentioned it:
`prepare` is not an arbitrary name, it is a `npm` built-in that [runs before many operations automatically](https://docs.npmjs.com/cli/v8/using-npm/scripts#life-cycle-scripts) - that is why `husky` uses it.

---

Maybe I should have been more explicit but this is what I meant to say that using `prepare` is special in that it runs when you run `bun install` and it is the same what happens when using [husky](https://typicode.github.io/husky/get-started.html#husky-init-recommended).